ironfish/src/consensus/consensus.ts

```diff
@@ -39,7 +39,7 @@ export type ConsensusParameters = {
   /**
    * The block height that enables the use of V2 transactions instead of V1
    */
-  enableAssetOwnership: number
+  enableAssetOwnership: number | null
```

Comment:
Nit: you can consider using a constant like `'never'` instead of `null` to make it even more explicit:

```
enableAssetOwnership: number | 'never'
```

Reply:
Thats good idea, the schema will be a little complicated but its doable. I will update.

Reply:
Updated PR